ddtrace/contrib/celery/util.py

```diff
@@ -0,0 +1,31 @@
+# Project
+from ddtrace import Pin
+
+# Service info
+APP = 'celery'
+SERVICE = 'celery'
+
+
+def meta_from_context(context):
+    """ helper to extract meta values from a celery context """
+    meta_keys = (
+        'called_directly', 'correlation_id', 'delivery_info', 'eta', 'expires', 'hostname',
```

Comment:
@clutchski curiosity, is there any particular reason to remove some of these tags?

Looking to save on bytes where we can, reduces processing, etc?